MAUI/DataForm/Overview.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Overview of .NET MAUI DataForm control | Syncfusion
+description: Overview of Syncfusion .NET MAUI DataForm(SfDataForm) control and its basic features.
+platform: Maui
+control: SfDataForm
+documentation: ug
+---
+ 
+ **SfDataForm**
+
+The Syncfusion .NET MAUI DataForm (SfDataForm) control is used to create or edit data form such as data form such as login, reservation, contact, employee form and so, based on the business purpose. 
+
+## Key features includes the following:
+
+*  **Layout and grouping**: Supports to linear and grid layout with grouping support. Supports customizing the layout with different heights for each item.
+
+
+* **Caption customization**: Supports loading the image as caption for the editor.
+
+
+* **Editors**: Built-in support for text, multiline text, password, picker, date picker, time picker, switch, comboBox ,autoComplete and checkbox editors. 
```

Comment:
add radio group

Reply:
replace contents adde by Jeyasri in above comment

Reply:
new content added